pandas/tests/frame/test_to_csv.py

```diff
@@ -874,6 +874,18 @@ def test_to_csv_index_no_leading_comma(self):
         expected = tm.convert_rows_list_to_csv_str(expected_rows)
         assert buf.getvalue() == expected
 
+    def test_to_csv_multi_index(self):
```

Comment:
we already have a multiindex test, can you move this near; change the test to be test_to_csv_single_level_multiindex

Reply:
Done in d47089ee3ef92152f9577602b84073455bb49763